docker/down.sh

```diff
@@ -22,14 +22,35 @@ project_root=$(git rev-parse --show-toplevel)
 cd "${project_root}/"
 
 compose_files="-f docker-compose.yml"
-args="--remove-orphans"
+compose_args="--remove-orphans"
+
+# Parse args
+while [ $# -gt 0 ]; do
+  case $1 in
+    -h|'--help')
+       usage
+       exit 0
+       ;;
+  esac
+  shift
+done
 
 # We can ignore the tag and port(s) when cleaning up running
 # containers and volumes
 TAG=any
 
-API_PORT=${RANDOM} API_ADMIN_PORT=${RANDOM} WEB_PORT=${RANDOM} TAG=${RANDOM} docker-compose $compose_files down $args && \
-  docker volume rm marquez_data && \
+API_PORT=${RANDOM}
+API_ADMIN_PORT=${RANDOM}
+WEB_PORT=${RANDOM}
+TAG=${RANDOM}
+
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+  docker compose ${compose_files} down ${compose_args}
+else
+  docker-compose $compose_files down $compose_args
```

Comment:
We can fully [migrate](https://docs.docker.com/compose/migrate/) off docker compose v1 to v2. That is, we can replace `docker-compose` with `docker compose` as docker compose v1 is no longer recieving updates after July 2023 (of this year).. This would also simplify the changes in the PR.